charts/index.yaml

```diff
@@ -12,7 +21,7 @@ entries:
     version: v2.0.0-beta.6
```

Comment:
We'll need to update the `latest-v2` chart to beta.7 as well.

Reply:
Oh, that's right!